src/lab11.py

```diff
@@ -587,12 +588,13 @@ def __init__(self):
 
     def handle_click(self, e):
         if e.y < self.chrome.bottom:
-            self.focus = "chrome"
+            self.focus = None
             self.chrome.click(e.x, e.y)
             self.raster_chrome()
         else:
             if self.focus != "content":
                 self.focus = "content"
+                self.chrome.focus = None
```

Comment:
Ooh. Does this actually mean we need to add blur methods to Chrome? Do we correctly handle clicking on the address bar while editing an input element? Or the reverse?

Reply:
I tested it pretty carefully and it works. I added a unittest though just now to further ensure correctness.